core/src/components/input/test/password-toggle/input.e2e.ts

```diff
@@ -0,0 +1,68 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, screenshot, config }) => {
+  test.describe(title('input: password toggle button'), () => {
+    test('should render password icon when show-password-toggle', async ({ page }) => {
```

Comment:
I was not able to commit the screenshots, so I am not sure if I should keep this test.

Reply:
Let's keep it for now. We have some work in review right now that will unblock you on this front.